src/component/project/ProjectCard/ProjectCard.tsx

```diff
@@ -104,9 +119,12 @@ export const ProjectCard = ({
                             e.preventDefault();
                             setShowDelDialog(true);
                         }}
+                        disabled={!canDeleteProject}
                     >
                         <Delete className={classes.icon} />
-                        Delete project
+                        {id === DEFAULT_PROJECT_ID && !canDeleteProject
+                            ? 'Cannot delete default project'
```

Comment:
```suggestion
                            ? 'You can't delete the default project'
```

I like this change a lot, but I think the wording could be improved a bit. How about this?

Also, will this be obvious to people with assistive tech? You can't focus the disabled button at least, but I'm not sure how that works with screen readers. I guess it announces the list?

Reply:
Good question. VoiceOver allows focus and announces the button as "dimmed", which seems appropriate.

Reply:
Right, yeah. That sounds about right to me, that. That means non-sighted and sighted users should both be able to see and read the text. So that strikes me as good :tm: